src/Analysim.Web/Controllers/ProjectController.cs

```diff
@@ -798,8 +798,26 @@ public async Task<IActionResult> DeleteProject([FromRoute] int projectID)
             var deleteProject = await _dbContext.Projects.FindAsync(projectID);
             if (deleteProject == null) return NotFound(new { message = "Project Not Found" });
 
-            // Remove Project
-            _dbContext.Projects.Remove(deleteProject);
+            // Remove all users that follow the project
+            foreach (var user in deleteProject.ProjectUsers)
+            {
+                _dbContext.ProjectUsers.Remove(user);
+            }
+
+            // Delete from Azure
+            var containerClient = _blobServiceClient.GetBlobContainerClient(deleteProject.Name.ToLower());
+            await containerClient.DeleteBlobIfExistsAsync(deleteProject.Name.ToLower());
```

Comment:
so why do you still have this one?

Reply:
I believe this deletes the Blobs inside the container but not the actual container itself